muikku-core-plugins/src/main/resources/META-INF/resources/jsf/records/index.xhtml

```diff
@@ -57,12 +58,44 @@
         </div>
       </ui:insert>
       <div class="mf-content-area">
-        <div>
-          <div class="tr-content-main flex-row" data-grades="#{transcriptofRecordsBackingBean.grades}">
-          </div>
-          <div class="tr-content-main flex-row" data-files="#{transcriptofRecordsBackingBean.files}">
+        <div class="tr-study-information">
+          <div class="tr-studydate-item">
+	          <ui:fragment rendered="#{!empty(transcriptofRecordsBackingBean.studyStartDate)}">
+	            <label class="tr-studydate-item-label">#{i18n.text['plugin.records.studyStartDateLabel']}</label>
+	            <span class="tr-studydate-item-data">
+	              <h:outputText value="#{transcriptofRecordsBackingBean.studyStartDate}">
+	                <f:convertDateTime type="date" dateStyle="short" />
+	              </h:outputText>                      
+	            </span>
+	          </ui:fragment>
           </div>
+          
+          <div class="tr-studydate-item">
+	          <ui:fragment rendered="#{!empty(transcriptofRecordsBackingBean.studyTimeEnd)}">
+	            <label class="tr-studydate-item-label">#{i18n.text['plugin.records.studyTimeEndLabel']}</label>
+	            <span class="tr-studydate-item-data">
+	              <h:outputText value="#{transcriptofRecordsBackingBean.studyTimeEnd}">
+	                <f:convertDateTime type="date" dateStyle="short" />
+	              </h:outputText>                      
+	              
+	              <ui:fragment rendered="#{(transcriptofRecordsBackingBean.studyTimeYearsLeft+transcriptofRecordsBackingBean.studyTimeMonthsLeft+transcriptofRecordsBackingBean.studyTimeDaysLeft) gt 0}">
+	                (
```

Comment:
A very minor (but annoying) issue. Would be nice if the opening bracket wasn't followed by a space and the closing bracket preceded by a space. Hard to achieve with multiple optional fragments in play, though. Of course, if the entire content with brackets and all would be just one string provided by the backing bean...